geometry_msgs/Point model
geometry_msgs/Pose measured